### PR TITLE
manifest: tfm: Apply hotfix for FOTA download

### PR DESCRIPTION
**Pull request by SebastianBoe**
Update TF-M manifest to apply hotfix of regression in nrf91 FOTA
download.

Fix from Joan Minana.

Signed-off-by: Sebastian Bøe <sebastian.boe@nordicsemi.no>

### CODE REVIEW

**Review by simensrostad — APPROVED** [review]
Tested with AWS FOTA, works. Thanks!

**Review by jhn-nordic — APPROVED** [review]
Seems to work on nRFCloud Fota as well

**Review by hakonfam — APPROVED** [review]

**Review by sigvartmh — APPROVED** [review]

**Comment by SebastianBoe** [comment]
waiting for CI before merging

**Pull request merged**
